ni_measurementlink_service/_internal/parameter/serialization_strategy.py

```diff
@@ -154,6 +219,7 @@ class Context:
         type_pb2.Field.TYPE_BOOL: (BoolEncoder, BoolArrayEncoder),
```

Comment:
I'm not sure what purpose this `Context` class serves. It seems to have only class data/methods, not instance data/methods. This is not C# or Java, so we don't need to wrap all data/methods in classes.